Documentation/ceph-upgrade.md

```diff
@@ -398,6 +398,8 @@ until all the daemons have been updated.
 ### **Ceph images**
 
 Official Ceph container images can be found on [Quay](https://quay.io/repository/ceph/ceph?tab=tags).
+Until recently (08/2021), official images were on docker.io. People using images from docker.io with the main major version tag (e.g. `v16`) will need to be changed to get the latest version.
```

Comment:
I hope nobody is using the tag `v16` in production, so how about a more general statement like this?
```suggestion
Prior to August 2021, official images were on docker.io. While those images will remain on dockerhub, all new images are being pushed to Quay.
```

Reply:
Me too 😓